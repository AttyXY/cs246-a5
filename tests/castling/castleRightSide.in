game human human
move e2 e3
move e7 e6
move f1 e2
move f8 e7
move g1 h3
move g8 h6
move e1 g1
move e8 g8