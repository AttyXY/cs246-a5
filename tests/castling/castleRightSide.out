WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING GAME!

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  --------
3  --------
2  PPPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  --------
3  ----P---
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  ----p---
5  --------
4  --------
3  ----P---
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  ----p---
5  --------
4  --------
3  ----P---
2  PPPPBPPP
1  RNBQK-NR
   abcdefgh

8  rnbqk-nr
7  ppppbppp
6  ----p---
5  --------
4  --------
3  ----P---
2  PPPPBPPP
1  RNBQK-NR
   abcdefgh

8  rnbqk-nr
7  ppppbppp
6  ----p---
5  --------
4  --------
3  ----P--N
2  PPPPBPPP
1  RNBQK--R
   abcdefgh

8  rnbqk--r
7  ppppbppp
6  ----p--n
5  --------
4  --------
3  ----P--N
2  PPPPBPPP
1  RNBQK--R
   abcdefgh

8  rnbqk--r
7  ppppbppp
6  ----p--n
5  --------
4  --------
3  ----P--N
2  PPPPBPPP
1  RNBQ-RK-
   abcdefgh

8  rnbq-rk-
7  ppppbppp
6  ----p--n
5  --------
4  --------
3  ----P--N
2  PPPPBPPP
1  RNBQ-RK-
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
